### ISSUE TITLE
cli_wallet get_account_history command returns incorrect result when limit is more than 100

### ISSUE THREAD

**Issue by abitmore** [cryptonomex/graphene#623]
I finally decided to write down this issue. Perhaps it's low priority, however, it seems that some parties have been affected by this issue.

According to the document, the command `get_account_history` is designed to **NOT work** with `limit` > 100, so strictly speaking it's a "feature".

```
/** Returns the most recent operations on the named account.
 *
 * This returns a list of operation history objects, which describe activity on the account.
 *
 * @note this API doesn't give a way to retrieve more than the most recent 100 transactions,
 *       you can interface directly with the blockchain to get more history
 * @param name the name or id of the account
 * @param limit the number of entries to return (starting from the most recent) (max 100)
 * @returns a list of \c operation_history_objects
 */
vector<operation_detail>  get_account_history(string name, int limit)const;
```

Currently, when calling this command with a number greater than 100 as `limit`, it will return more than 100 records if there are more than 100 records with that account, among them, the first 100 records are correct, and the rest record(s) is/are usually incorrect. It's confusing. In my opinion, maybe it's better to refuse to execute the command for example throw out an exception if given a `limit` greater than 100, or just return the first 100 records only.

**Comment by theoreticalbts** [comment]
There is at least one related bug fixed in #477 branch (which will be merged shortly).

The comment doesn't match the code.  The code appears to attempt to build a non-paginated API from the paginated API provided by the `witness_node`.

**Comment by abitmore** [comment]
It seems the issue has been fixed in `2.0.160316` tag. It now can be used to fetch more than 100 records correctly. 

The [in-code comments](https://github.com/bitshares/bitshares-2/blob/2.0.160316b/libraries/wallet/include/graphene/wallet/wallet.hpp#L327-L337) need to be revised, since it's used to generate documents via Doxygen.

By the way, current implementation of `get_account_history` command is inefficient when `limit` is large, but the optimization probably belongs to another ticket.

**Comment by vikramrajkumar** [comment]
This issue was moved to bitshares/bitshares-core#193
